src/components/Main/Recommendation.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import RecommendedBook from './RecommendedBook';
+
+export class Recommendation extends Component {
+  constructor() {
+    super();
+    this.state = {
+      NewIssueList: [],
+      startIndex: 0,
+      endIndex: 5,
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/NewIssueList.json', {
+      method: 'GET',
+    })
```

Comment:
```suggestion
    fetch('/data/NewIssueList.json')
```
default 값을 이용한다면 이렇게 줄일 수 있습니다!

Reply:
안내해주신 방식대로 수정했습니다! Bestseller.js 파일에서도 동일하게 적용하겠습니다 : )